simple.js

```diff
@@ -496,6 +498,48 @@ define(function(require, exports, module) {
         }
 
         /**
+         * Detects Ace syntax mode from shebang for files with no extension
+         */
+        function detectSyntax() {
+            // syntax modes
+            var MODES = {
+                perl: /perl/,
+                php: /php/,
+                python: /python/,
+                ruby: /ruby/,
+                sh: /^(bash2?|ksh|sh)/
```

Comment:
Well, C9's syntax mode is named [`sh`](https://github.com/c9/core/blob/master/node_modules/ace/lib/ace/mode/sh.js). So I only took `sh` along with everything else that vim groups with it.